doc/man/8/radosgw.rst

```diff
@@ -68,47 +68,51 @@ Configuration
-traditional FastCGI module. This module requires the service of ``mod_proxy``
-which provides support for the FastCGI protocol. So, to be able to handle
-FastCGI protocol, both ``mod_proxy`` and ``mod_proxy_fcgi`` have to be present
-in the server. Unlike ``mod_fastcgi``, ``mod_proxy_fcgi`` cannot start the
-application process. Some platforms have ``fcgistarter`` for that purpose.
-However, external launching of application or process management may be available
-in the FastCGI application framework in use.
+Now, ``mod_proxy_fcgi`` module is used instead of ``mod_fastcgi``.
+``mod_proxy_fcgi`` works differently than a traditional FastCGI module. This
+module requires the service of ``mod_proxy`` which provides support for the
+FastCGI protocol. So, to be able to handle FastCGI protocol, both ``mod_proxy``
+and ``mod_proxy_fcgi`` have to be present in the server. Unlike ``mod_fastcgi``,
+``mod_proxy_fcgi`` cannot start the application process. Some platforms have
+``fcgistarter`` for that purpose. However, external launching of application
+or process management may be available in the FastCGI application framework
+in use.
 
 ``Apache`` can be configured in a way that enables ``mod_proxy_fcgi`` to be used
 with localhost tcp or through unix domain socket. ``mod_proxy_fcgi`` that doesn't
 support unix domain socket such as the ones in Apache 2.2 and earlier versions of
-Apache 2.4, needs to be configured for use with localhost tcp.
+Apache 2.4, needs to be configured for use with localhost tcp. Later versions of
+Apache like Apache 2.4.9 or later support unix domain socket and as such they
+allow for the configuration with unix domain socket instead of localhost tcp.
 
-#. Modify ``/etc/ceph/ceph.conf`` file to make radosgw use tcp instead of unix
-   domain socket. ::
+The following steps show the configuration in Ceph's configuration file i.e,
+``/etc/ceph/ceph.conf`` and the gateway configuration file i.e,
+``/etc/httpd/conf.d/rgw.conf`` with localhost tcp and through unix domain socket:
 
-	[client.radosgw.gateway]
-	host = gateway
-	keyring = /etc/ceph/keyring.radosgw.gateway
+#. For distros with Apache 2.2 and early versions of Apache 2.4 that use
+   localhost TCP and do not support Unix Domain Socket, append the following
+   contents to ``/etc/ceph/ceph.conf``::
 
-	; ********
-	; tcp fastcgi
-		rgw socket path = ""
-		rgw frontends = fastcgi socket_port=9000 socket_host=0.0.0.0
+	[client.radosgw.gateway]
+	host = {hostname}
+	keyring = /etc/ceph/ceph.client.radosgw.keyring
+	rgw socket path = ""
+	log file = /var/log/radosgw/client.radosgw.gateway.log
+	rgw frontends = fastcgi socket_port=9000 socket_host=0.0.0.0
+	rgw print continue = false
 
-#. Modify Apache's configuration file so that ``mod_proxy_fcgi`` can be used
-   with localhost tcp.
+#. Add the following content in ``/etc/httpd/conf.d/rgw.conf``:
 
    Debian/Ubuntu::
 
-		<VirtualHost *:80>
+		<VirtualHost {IP ADDRESS}:80>
```

Comment:
It's actually ok to have the `<VirtualHost *:80>` directive use "`*`" instead of "`{IP ADDRESS}`". It simplifies the configuration if users don't have to enter their IP address here.